### ISSUE TITLE
Order of inputs in .pf file.  Disk spectrum type is requested before we know there is adisk

### ISSUE THREAD

**Issue by kslong** [sirocco-rt/sirocco#8]
Currently we ask whether to include a disk spectrum and its type before determine that a model has a disk.  We need to change the order of the inputs, but this will mean all .pf files need to be changed.   It's likely we should make sure we have reviewed the .pf files, to see if there is anything else we need to change before updating python.c

**Comment by jhmatthews** [comment]
Proposal here: move disk_type up to just after system type.

```
System_type(0=star,1=binary,2=agn)                   2
disk.type(0=no.disk,1=standard.flat.disk,2=vertically.extended.disk)                    1
Disk_radiation(y=1)                               1
Wind_radiation(y=1)                               1
QSO_BH_radiation(y=1)                             1
Rad_type_for_disk(0=bb,1=models)_to_make_wind                    0
Rad_type_for_agn(0=bb,1=models,3=power_law,4=cloudy_table)_to_make_wind                   3
mstar(msol)                                    1e7
rstar(cm)                                 1.47611e+12
# old position: disk.type(0=no.disk,1=standard.flat.disk,2=vertically.extended.disk)                    1
disk.mdot(msol/yr)                         8.8267e-3
Disk.illumination.treatment(0=no.rerad,1=high.albedo,2=thermalized.rerad,3=analytic)                    0
Disk.temperature.profile(0=standard;1=readin)                    0
disk.radmax(cm)                               5e16
```

parameter files will need to be changed accordingly.

**Comment by jhmatthews** [comment]
Closed by #139
